package.json

```diff
@@ -10,8 +10,9 @@
   ],
   "types": "src/index.d.ts",
   "scripts": {
-    "lint": "aegir lint",
+    "lint": "eslint --ext .ts ts",
     "release": "aegir release",
+    "prebuild": "tsc",
     "build": "aegir build",
     "test": "aegir test",
```

Comment:
shoul we add the prebuild step to the test script? Otherwise, it will fail until we run it.

> open question - should src/ be checked into git? right now it is not

This would solve this issue. But this is not expected in a typescript project, right?

Reply:
Yeah, good idea, will add a `pretest` script. In the PR currently, I've manually run `prebuild` before each test step in travis, not a very good solution.

In adding `src/` to version control, I'd just be worried about keeping it in sync. I think that's one reason its not checked in, similar to how we don't check in the `dist/` folder